eo-maven-plugin/src/test/java/org/eolang/maven/PullMojoTest.java

```diff
@@ -238,36 +242,40 @@ void pullsProbedVersionedObjectsFromDifferentObjectionaries(@TempDir final Path
             .with("hsh", third)
             .withVersionedProgram()
             .execute(new FakeMaven.Pull());
-        final String sprintf = "%s/org/eolang/io/sprintf|17f892.eo";
-        final String string = "%s/org/eolang/string|5f82cc";
+        final ObjectName sprintf = new OnCached(
```

Comment:
@maxonfjvipon You are using the same pattern through all the places where you need `ObjectName`:
```java
final ObjectName string = new OnCached(
   new OnDefault(
       "%s/org/eolang/string", 
       new CommitHash.ChConstant("5f82cc.eo")
    )
);
```
What does mean that code? Right:
```
"%s/org/eolang/string|5f82cc"
```
That is all. The code which we have looks rather overcomplicated and verbose.  Don't you think so? I understand, that is already better than it was, but can we simplify this code somehow? 
I don't know how we can achieve that, but probably, we can:
1. Avoid caching in tests. It doesn't give any significant performance (at least it seem so), hence we can sacrifice it.
2. Create one more constructor or even other object which will look much simpler:
```java
new ObjectName("%s/org/eolang/string", "5f82cc");
```